rest/tests/test_learning_resource.py

```diff
@@ -104,6 +107,34 @@ def test_missing_learning_resource(self):
                                    expected_status=HTTP_404_NOT_FOUND)
         self.get_learning_resource(repo_slug2, lr2_id)
 
+    def test_learning_resource_filter(self):
+        """Test for filtering learning resources by ids"""
+
+        def get_filtered(ids, expected_status=HTTP_200_OK):
+            """Return list of learning resourceswhich are in shopping cart"""
```

Comment:
Typo: "resourceswhich" 

Also, maybe sentence should just be:

"Return list of LearningResources in shopping cart."